homepage.css

```diff
@@ -0,0 +1,79 @@
+@import url("https://fluid-ui-component.vercel.app/docs/docs.css");
+@import "/style.css";
+.category-container {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  padding: 1rem 0;
+  width: 100%;
+}
+.banner-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  padding: 2rem;
+  width: 98%;
+  border: 2px solid black;
+  margin: 1%;
```

Comment:
use rem unit

Reply:
not using rem since its a very small unit, it wont affect